src/Employer.Web/ViewModels/Preview/IndexViewModel.cs

```diff
@@ -8,5 +8,7 @@ namespace Esfa.Recruit.Employer.Web.ViewModels.Preview
     public class IndexViewModel
     {
         public string Title { get; set; }
+
+        public bool IsSubmittable { get; set; }
```

Comment:
Probably call this `CanSubmit` maybe, save some keystrokes 😉

Reply:
I agonised over this. I originally called it CanSubmit but tried squeeze an 'Is' on to the start of it.  I'm happier with CanSubmit if you are - thought you were more of an 'Is' person. Will change

Reply:
Thanks for considering me a person. Not too bothered but thought about your keystrokes reasoning.